### PR TITLE
Multi channel support

### PR DESCRIPTION
**Pull request by inzamam-iqbal**
This PR will work only after merging the [PR in helm stable repo ](https://github.com/helm/charts/pull/15321)

fixes: #46

### CODE REVIEW

**Comment by alexvicegrab** [comment]
Happy to review once we merge the `helm/charts` update to the Peer (hopefully later today), as soon as small fixes are done as per my review there: https://github.com/helm/charts/pull/15321

File: examples/prod/nephos_config.yaml [hunk 1]
```diff
@@ -1,8 +1,8 @@
 core:
   # Comment out to make it work on your cluster
-  cluster: minikube
+  # cluster: minikube
   # ChartMuseum repository or directory holding charts
-  chart_repo: stable
+  chart_repo: aidtech
```
Comment (alexvicegrab) [inline examples/prod/nephos_config.yaml]: This should be `stable`, even if we test locally on the `aidtech` repo.

File: nephos/fabric/peer.py [hunk 2]
```diff
@@ -223,7 +223,7 @@ def create_channel(opts):
                             (
                                 "bash -c 'peer channel create "
                                 + f"-o {ord_name}-hlf-ord.{ord_namespace}.svc.cluster.local:7050 "
-                                + f"-c {channel_name} -f /hl_config/channel/{channel_name}.tx {cmd_suffix}'"
+                                + f"-c {channel_name} -f /hl_config/channel/{channel_name}/{channel_name}.tx {cmd_suffix}'"
```
Comment (alexvicegrab) [inline nephos/fabric/peer.py]: Aha, in new Helm chart each Channel Tx is located to its own folder?

**Comment by alexvicegrab** [comment]
In the meantime I request one change here.

File: nephos/fabric/peer.py (hunk 2 above)
Comment (inzamam-iqbal) [reply nephos/fabric/peer.py]: yes

**Comment by alexvicegrab** [comment]
@inzamam-iqbal, please update the integration tests so that they display output and we can debug.

**Comment by inzamam-iqbal** [comment]
@alexvicegrab tests passes now. I will put a PR to update the integration test log

**Pull request merged**